sympy/physics/mechanics/tests/test_jointsmethod.py

```diff
@@ -0,0 +1,87 @@
+from sympy import Symbol, symbols, ImmutableMatrix as Matrix
+from sympy.physics.mechanics import *
+
+
+def test_joint_method_simple():
+    parent = Body('parent')
+    child = Body('child')
+    joints = []
+    gravity = Symbol('gravity')
+    child.apply_force(gravity * parent.frame.y)
+    pin_joint = PinJoint('pinjoint', parent, child)
+    joints.append(pin_joint)
+
+    JM = JointsMethod(joints, parent)
+    assert JM.rhs() == Matrix([
+        [pin_joint.speeds[0]], [0]])
```

Comment:
This seems incorrect. The rhs for a simple pendulum with both mass an inertia of the body should be something like:
m * g * l * sin(theta) / (m * l**2 + I)